executor/executor_bsd.h

```diff
@@ -66,6 +66,13 @@ static void cover_open(cover_t* cov, bool extra)
 	unsigned long cover_size = kCoverSize;
 	if (ioctl(cov->fd, KIOSETBUFSIZE, &cover_size))
 		fail("ioctl init trace write failed");
+	if (extra) {
+		struct kio_remote_attach args;
+		args.subsystem = KCOV_REMOTE_COMMON;
+		args.id = 0;
```

Comment:
Shouldn't this use procid somewhere? Without passing procid to the kernel, I am not sure it can possibly work.
The point of remote coverage is to associate coverage from a background kernel thread/interrupt with a particular syzkaller test process N. If there is something happens in a background thread, kernel needs to know what is that associated N to attribute that coverage to.

Also I assume some kernel annotations were added as well for some subsystems, right? Without kernel annotations remote coverage won't collect anything. At least that's how it works on linux and how it's supposed to be used.

@xairy